LICENSE

```diff
@@ -1,201 +1,674 @@
-                                 Apache License
```

Comment:
I had one question about the license. (Can we just change it like this?)

I changed it to GPL v3 since that's the proposed license we have in the odh-community PR Sherard raised.

Reply:
I think it is safe to migrate in this commit since this is brand new code that will be covered by GPL

Reply:
Not aware of anything against changing the license.  Previously, when we were derivative from upstream's kubeflow operator, it may have been an issue, but since this is all-new, I think we should use the odh-community standard license

Reply:
Sweet. Thanks folks.